src/language/es/es.json

```diff
@@ -99,10 +99,11 @@
     "pairSupply": "Circulante",
     "estimatedPrice": "Precio",
     "btnLabel": "Operar",
-    "price": "Precio"
+    "price": "Precio",
+    "yourWallet": "Su billetera"
```

Comment:
Should there be a capital B? in `billetera` ?

Reply:
I think so, I update it in a new commit 👍